tools/ci_build/github/azure-pipelines/web-ci-pipeline.yml

```diff
@@ -1,3 +1,45 @@
+trigger:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    include:
+    - 'js/web'
+    - 'js/node'
+    - 'onnxruntime'
+    - 'cmake'
+    - 'include'
+    - 'tool'
+    - '.gitmodules'
+    - 'setup.py'
+    - 'build.*'
+    - '.github'
+    exclude:
+    - 'onnxruntime/core/providers/tensorrt'
+    - 'onnxruntime/core/providers/cuda'
+    - 'onnxruntime/core/providers/rocm'
+pr:
+  branches:
+    include:
+    - main
+    - rel-*
+  paths:
+    include:
+    - 'js/web'
+    - 'js/node'
+    - 'onnxruntime'
+    - 'cmake'
+    - 'include'
+    - 'tool'
+    - '.gitmodules'
+    - 'setup.py'
+    - 'build.*'
+    - '.github'
+    exclude:
+    - 'onnxruntime/core/providers/tensorrt'
+    - 'onnxruntime/core/providers/cuda'
+    - 'onnxruntime/core/providers/rocm'
```

Comment:
Please add a task that will delete the unwanted files at the beginning of the build jobs.
The inclusion/exclusion rule here is very complicated. Without some validation, it could soon go out of control.

Reply:
Will create a separate PR for web-ci-pipeline.yml after clean out files